documentation/dev/docker-compose.rst

```diff
@@ -0,0 +1,80 @@
+.. _docker-compose:
+
+Running a complete stack with docker-compose
+=============================================
+
+Installing FlexMeasures and dependencies is some work, and can have unexpected hurdles, e.g. depending on the operating system. So, running the whole stack via `Docker compose <https://docs.docker.com/compose/>`_ can be a very useful option.
```

Comment:
If you re-build the FlexMeasures image from code, don't you still have to install FlexMeasures and its dependencies? If so, the introductory line is confusing to me.

Reply:
Maybe by 'dependencies' here you meant other docker images? Isn't docker-compose just about packaging an application that involves multiple docker containers?

Reply:
I meant that doing this yourself is a hassle - docker compose does it for you. I can probably improve that wording.